src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -274,6 +274,16 @@ public async Task<ResourceConfig<VirtualMachineScaleSet>> CreateConfigAsync()
 
             private async Task<ResourceConfig<VirtualMachineScaleSet>> SimpleParameterSetNormalMode()
             {
+                // Temporary message until after the Ignite 2023 release that should remove these outdated image aliases. 
+                if ((_cmdlet.ImageName != "")
```

Comment:
`(_cmdlet.ImageName != "")` is not necessary unless most of the `ImageName` is empty.

Reply:
It is possible that the ImageName is empty because it is an optional parameter. This check should be here I think.

Reply:
@Sandido if the `ImageName` is empty, then it won't equal to "CentOS" | "Debian" | "RHEL" | "UbuntuLTS". So if most of the cmdlets would receive an empty string for this parameter, then it is fair enough to verify it at the beginning to save time. Otherwise, it is not that necessary.

Reply:
@Sandido any update?

Reply:
@Sandido What do you think of this comment?

Reply:
@vidai-msft , In the past 90 days in kusto, 
about 30% of New-AzVM calls that can have -Image (SimpleParameterSet) do not use it so it would be empty.
About 68% of New-AzVmss calls that can have -ImageName do not use it so it would be empty.